dashboard/src/locales/fr.json

```diff
@@ -60,52 +60,75 @@
   "FAQQ17": "Êtes-vous financé ?",
   "FAQA17": "En ce moment, nous avons <a href=\"https://kusama.polkassembly.io/post/384\">Kusama Treasury funding for Milestone 1</a>. Nous recherchons activement des options de financement, nous pourrions éventuellement lever des fonds privés. DMs nous, si vous êtes intéressés par une vente privée.",
-  "FAQQ18": "Avez-vous Télégramme canal?",
-  "FAQA18": "Nous n'avons qu'une chaîne privée pour les créateurs NFT, DM nous si vous êtes créateur NFT !",
+  "FAQQ18": "Avez-vous un canal Telegram ?",
+  "FAQA18": "Nous avons seulement un canal privé pour les créateurs de NFT, DM nous si tu es créateur de NFT !",
   "FAQQ19": "Allez-vous avoir le mode AR/VR ou la mise en œuvre ?",
   "FAQA19": "Oui, cela fait partie de notre étape 2, restez à l'écoute !",
   "FAQQ20": "Comment ouvrir la console du développeur ?",
   "FAQA20": "Si vous êtes sur Mac, sur Chrome tapez <b>Option, CMD, J</b>, Firefox <b>Command + Option + K</b>.",
   "FAQQ21": "Comment mettre à jour les métadonnées pour l'extension ?",
-  "FAQA21": "Allez voir <a href=\"https://cloudflare-ipfs.com/ipns/dotapps.io/?rpc=wss%3A%2F%2Fkusama-rpc.polkadot.io#/settings/metadata\">Dotapps.io, Settings > Metadata</a>  et voyez là le bouton <b>Mettre à jour les métadonnées</b>, puis rafraîchissez votre fenêtre, devrait fonctionner avec les types de chaîne récents.",
-  "FAQQ22": "['J'ai une \"erreur de solde faible",
-  "FAQA22": "Vérifiez si vous avez des $KSM <b>transférables</b> sur votre compte, non mis en jeu. C'est différent. Vous pouvez le vérifier dans  <a href=\"https://cloudflare-ipfs.com/ipns/dotapps.io/?rpc=wss%3A%2F%2Fkusama-rpc.polkadot.io#/accounts\">Dotapps.io > Comptes<a/> et à balances déplier, vous pouvez voir là montant <b>transférable</b>"
+  "FAQA21": "Allez voir sur <a href=\"https://cloudflare-ipfs.com/ipns/dotapps.io/?rpc=wss%3A%2F%2Fkusama-rpc.polkadot.io#/settings/metadata\">Dotapps.io, Settings > Metadata</a>  et voyez là le bouton <b>Mettre à jour les métadonnées</b>, puis rafraîchissez votre fenêtre, devrait fonctionner avec les types de chaîne récents.",
+  "FAQQ22": "J'ai une erreur \"Low balance error\"",
+  "FAQA22": "Vérifiez si vous avez des $KSM <b>transférables</b> sur votre compte, non mis en jeu. C'est différent. Vous pouvez le vérifier dans  <a href=\"https://cloudflare-ipfs.com/ipns/dotapps.io/?rpc=wss%3A%2F%2Fkusama-rpc.polkadot.io#/accounts\">Dotapps.io > Comptes<a/> et à balances déplier, vous pouvez voir là montant <b>transférable</b>",
+  "FAQQ23": "Comment définir l'identité ?",
+  "FAQA23": "Vous pouvez en savoir plus sur <a href=\"https://guide.kusama.network/docs/en/mirror-learn-identity#docsNav\">comment définir l'identité sur le réseau Kusama</a>",
+  "sort": {
```

Comment:
I think we will have later different translate system but doesn't support nested stuff, I can recommend to put it there in way like sort_created and rework commit like that because now builds fails probably on this :)

I can check it later this week if you don't figure out on yourselves.

Reply:
I took as example the en.json and rework fr.json file.

Build/Serve is ok to me and nested translation worked well (see screenshot)
![Screenshot 2021-06-02 at 08-51-01 Low minting fees and carbonless NFTs](https://user-images.githubusercontent.com/9987732/120436807-afa4ad00-c37f-11eb-8e44-8064d1bcb0a5.png)

Should we wait for #284 to push new content ?

Reply:
I mean #284 is quite a stall, so if you feel you can rework it, it's up to you! Bounty is still there :)